.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+-   repo: https://gitlab.com/pycqa/flake8
```

Comment:
Well, I use that repo cuz when I was looking at the hooks supported from pre-commit-hooks I found that repo on their page.
https://pre-commit.com/hooks.html
But I can change it if u think so??

Reply:
Not really, just idle curiosity. If pre-commit version had some kind of `stable` tag that we could follow, it would make sense to use it, but it doesn't :)